zanata-war/src/main/webapp/version-group/version_group.xhtml

```diff
@@ -215,19 +215,13 @@
                 role="content"
                 aria-labelledby="dropdownContent">
                 <li>
-                  <s:link view="/version-group/request_to_join.xhtml"
-                    styleClass="i__item--right"
-                    propagation="none">
-                    <f:param name="emailType"
-                      value="request_to_join_group"/>
-                    <f:param name="versionGroupSlug"
-                      value="#{versionGroupHomeAction.slug}"/>
-                    <h:outputFormat
-                      value="#{msgs['jsf.RequestAddProjectToGroup']}">
-                      <f:param value="#{versionGroupHome.instance.name}"/>
-                    </h:outputFormat>
+                  <a href="#" class="i__item--right"
+                    data-toggle="modal" onclick="return false;"
+                    data-target="#joinGroupDialog">
+                    #{msgs.format('jsf.RequestAddProjectToGroup', versionGroupHome.instance.name)}
                     <i class="i i--project i__item__icon"></i>
-                  </s:link>
+                  </a>
+                  <ui:include src="../WEB-INF/layout/version-group/request_join_modal.xhtml"/>
```

Comment:
This should not need to use `../WEB-INF`, should work with just `/WEB-INF` and will be more robust.